### DISCUSSION TITLE
Custom css classes are missing in build file

### DISCUSSION THREAD

**Discussion by marufdevops** [tailwindlabs/tailwindcss#7305] [Help] [answered]
**What version of Tailwind CSS are you using?**
3.0.17
**What build tool (or framework if it abstracts the build tool) are you using?**
Laravel Mix
**What version of Node.js are you using?**
v14.17.0
**What browser are you using?**
Chrome
**What operating system are you using?**
macOS

**Describe your issue**
After running the build script it seems like some of the custom classes are missing in public css file. Not sure why this is happening!
 
 For example: 
 .form-label appears in public css file
 ```
.form-label {
  @apply block text-sm font-normal text-black;
}
```

but .form-label-error does not!

```
.form-label-error {
  @apply block text-sm font-normal text-red;
}

```

This is how my custom css file look in general: 
![Screen Shot 2022-02-02 at 6 51 23 PM](https://user-images.githubusercontent.com/56079318/152157235-7b2223c5-8439-4f15-9a7d-d2f999ed3344.png)

This is how my mix config is:
![Screen Shot 2022-02-02 at 6 52 06 PM](https://user-images.githubusercontent.com/56079318/152157340-7ed1af2b-86ea-4edc-a56f-2cc8d20c0e86.png)

**Comment by adamwathan** [comment, marked as answer]
Are you making sure that `form-label-error` is present in your template files that match your content configuration? Tailwind will only generate that class if it sees you using it since you are putting it in one of Tailwind’s layers.

If you create a proper reproduction project it will be a lot easier to help.

Also — those custom classes look like components to me not utilities, I would recommend putting them in the components layer.

**Reply by marufdevops** [reply]
Thank you, it helped!
